src/systems/systems/src/hydraulic/rudder_control.rs

```diff
@@ -0,0 +1,1137 @@
+        fn update_before_power_distribution(
+            &mut self,
+            _: &UpdateContext,
+            electricity: &mut Electricity,
+        ) {
+            self.powered_source_ac
+                .power_with_potential(ElectricPotential::new::<volt>(140.));
+            electricity.supplied_by(&self.powered_source_ac);
+
+            if self.is_nominal_elec_powered {
+                electricity.flow(&self.powered_source_ac, &self.dc_2_bus);
+                electricity.flow(&self.powered_source_ac, &self.dc_ess_bus);
+            }
+
+            if self.is_emergency_elec_powered {
+                electricity.flow(&self.powered_source_ac, &self.ac_ess_bus);
+                electricity.flow(&self.powered_source_ac, &self.ac_2_bus);
+            }
+        }
+
+        fn update_after_power_distribution(&mut self, context: &UpdateContext) {
+            self.updater_fixed_step.update(context);
+
+            for cur_time_step in &mut self.updater_fixed_step {
+                self.rudder_control.update(
+                    &context.with_delta(cur_time_step),
+                    self.rudder_pedal_input,
+                    &self.trim_controller,
+                    &self.limiter_controller,
+                    &self.yaw_damper_controllers,
+                    self.hydraulic_pressures,
+                );
+
+                println!(
+                    "TRIM DEMAND: {:.3} ,TRIM REAL POS {:.3}, LIMITER +/- {:.3} Ydamper {:.3} Final actuator input {:.3} Flows gpm G Y {:.2} {:.2}",
+                    self.trim_controller.commanded_position()[0].get::<degree>(),
+                    self.rudder_control.trim.mechanism.angle.get::<degree>(),
+                    self.rudder_control.travel_limiter.mechanism.angle.get::<degree>(),
+                    self.rudder_control.yaw_damper.angle().get::<degree>(),
+                    self.rudder_control.final_actuators_input.get::<degree>(),
+                    self.rudder_control.yaw_damper.actuators[0].flow.get::<gallon_per_minute>(),
+                    self.rudder_control.yaw_damper.actuators[1].flow.get::<gallon_per_minute>()
+                );
+            }
+        }
+    }
+    impl SimulationElement for TestAircraft {
+        fn accept<V: SimulationElementVisitor>(&mut self, visitor: &mut V) {
+            self.rudder_control.accept(visitor);
+
+            visitor.visit(self);
+        }
+    }
+
+    #[test]
+    fn trim_demand_up_15_degrees_then_down() {
+        let mut test_bed = SimulationTestBed::new(TestAircraft::new);
+
+        test_bed.command(|a| {
+            a.set_trim_demand(
+                [true, false],
+                [Angle::new::<degree>(15.), Angle::new::<degree>(15.)],
+            )
+        });
+        test_bed.run_with_delta(Duration::from_millis(17000));
+
+        let trim_deflection: Angle = test_bed.read_by_name("HYD_RUDDER_TRIM_FEEDBACK_ANGLE");
+        assert!((trim_deflection.get::<degree>() - 15.).abs() < 0.1);
```

Comment:
```suggestion
        assert_about_eq!(trim_deflection.get::<degree>(), 15., 0.1);
```
Maybe consider using `assert_about_eq!`. It makes things easier to read IMO.
Just for the future.